recipe/meta.yaml

```diff
@@ -38,6 +38,7 @@ outputs:
         - setuptools
         - setuptools_scm >=4
```

Comment:
Should be `setuptools_scm >=7`, see https://github.com/matplotlib/matplotlib/blob/318b2348f7127efc3e28aadebf762aae32b6343c/setup.py#L306